src/main/topLevelPages/myDashboard/releaseNotesBanner/ReleaseNotesBanner.tsx

```diff
@@ -1,27 +1,38 @@
 import { Trans, useTranslation } from 'react-i18next';
+import CampaignOutlinedIcon from '@mui/icons-material/CampaignOutlined';
 import { BlockTitle, Caption } from '../../../../core/ui/typography';
 import React from 'react';
 import DashboardBanner from '../../../../core/ui/content/DashboardBanner';
 import useReleaseNotes from '../../../../domain/platform/metadata/useReleaseNotes';
+import RouterLink from '../../../../core/ui/link/RouterLink';
+import { useConfig } from '../../../../domain/platform/config/useConfig';
+
+const URL_NOT_CLICKABLE_MARKER = 'NOT_CLICKABLE';
```

Comment:
I think this is a bit hacky, we can just put a boolean value in the translation the product team will understand that better if they need to change it. `"isLink": false` in thr translation file makes `const isLink = t('releaseNotes.isLink');` a boolean too that we can use

Reply:
It makes sense, thanks @ccanos . I'll update the implementation to depend on a separate flag. isLink with a string value as I'm not sure how boolean would be treated in Crowdin.